tests/end_to_end/test_slack_integration/test_slack_integration.py

```diff
@@ -0,0 +1,100 @@
+import os
+import json
+import re
+import pytest
+from tempfile import gettempdir
+
+import wazuh_testing as fw
+from wazuh_testing.tools.file import remove_file
+from wazuh_testing import end_to_end as e2e
+from wazuh_testing import event_monitor as evm
+from wazuh_testing.tools import configuration as config
+
+# Test cases data
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+test_cases_path = os.path.join(test_data_path, 'test_cases')
+test_cases_file_path = os.path.join(test_cases_path, 'cases_slack_integration.yaml')
+alerts_json = os.path.join(gettempdir(), 'alerts.json')
+
+# Playbooks
+configuration_playbooks = ['configuration.yaml']
+events_playbooks = ['generate_events.yaml']
+teardown_playbooks = ['teardown.yaml']
+
+# Configuration
+configuration, metadata, cases_ids = config.get_test_cases_data(test_cases_file_path)
+
+# Custom paths
+slack_api_script = os.path.join(test_data_path, 'configuration', 'slack_api_script.py')
+
+# Update configuration with custom paths
+metadata = config.update_configuration_template(metadata, ['CUSTOM_SLACK_SCRIPT_PATH'], [slack_api_script])
+
+
+@pytest.fixture(scope='function')
+def get_slack_log_path():
```

Comment:
The path should be defined above and the fixture should only delete the file.

Reply:
Done in 29a7187